projects/nutanix-cloud-native/cluster-api-provider-nutanix/expected_artifacts

```diff
@@ -0,0 +1,18 @@
+SHA256SUM
+SHA256SUM.sha256
+SHA256SUM.sha512
+SHA512SUM
+SHA512SUM.sha256
+SHA512SUM.sha512
+cluster-api-provider-nutanix-linux-amd64-$GIT_TAG.tar.gz
+cluster-api-provider-nutanix-linux-amd64-$GIT_TAG.tar.gz.sha256
+cluster-api-provider-nutanix-linux-amd64-$GIT_TAG.tar.gz.sha512
+cluster-api-provider-nutanix-linux-arm64-$GIT_TAG.tar.gz
+cluster-api-provider-nutanix-linux-arm64-$GIT_TAG.tar.gz.sha256
+cluster-api-provider-nutanix-linux-arm64-$GIT_TAG.tar.gz.sha512
+githash
+manifests/infrastructure-nutanix/$GIT_TAG/cluster-template.yaml
+manifests/infrastructure-nutanix/$GIT_TAG/infrastructure.cluster.x-k8s.io_nutanixclusters.yaml
+manifests/infrastructure-nutanix/$GIT_TAG/infrastructure.cluster.x-k8s.io_nutanixmachines.yaml
+manifests/infrastructure-nutanix/$GIT_TAG/infrastructure.cluster.x-k8s.io_nutanixmachinetemplates.yaml
```

Comment:
This should also be part of `infrastructure-components.yaml` instead of individual CRD yamls